.github/workflows/concrete_python_release.yml

```diff
@@ -58,10 +58,11 @@ jobs:
       - name: Set release version (nightly)
         if: ${{ env.RELEASE_TYPE == 'nightly' }}
         run: |
-          VERSION=$(date +"%Y.%m.%d")
-          echo "__version__ = \"$VERSION\"" >| frontends/concrete-python/version.txt
-          git tag nightly-$VERSION || true
-          git push origin nightly-$VERSION || true
+          NIGHTLY_VERSION=$(date +"%Y.%m.%d")
+          LATEST_RELEASE_VERSION=`git describe --tags --abbrev=0 --match="v*" | grep -e '[0-9].*' -o`
+          echo "__version__ = \"${LATEST_RELEASE_VERSION}-dev${NIGHTLY_VERSION}\"" >| frontends/concrete-python/version.txt
```

Comment:
${NIGHTLY_VERSION}-dev is not enough?
with pip install --pre it will be overlapping with rc but in that particular case you could still ask user to specify the version manually no?

Reply:
it can be enough yes, but the idea of having this big MAJOR value seems off to me. This format makes it clear where a certain nightly belongs.

Reply:
make sense